src/main/features/dashboard/views/macro/claimStatus/claimFullAdmission.njk

```diff
@@ -0,0 +1,116 @@
+{% from "externalLink.njk" import externalLink %}
+{% from "form.njk" import linkButton %}
+
+{% macro claimFullAdmissionForDefendantDashboard(claim) %}
+  {% if claim.response.paymentIntention.paymentDate %}
```

Comment:
Please remove this and redo inside `claim.response.paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE` block. In that scenario 'if' statement is not needed.